java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -744,6 +744,45 @@ public void sadd_srem_scard_smembers_key_with_non_set_value(BaseClient client) {
         assertTrue(e.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void smove(BaseClient client) {
+        String setKey1 = "{key}" + UUID.randomUUID();
+        String setKey2 = "{key}" + UUID.randomUUID();
+        String setKey3 = "{key}" + UUID.randomUUID();
+        String nonSetKey = "{key}" + UUID.randomUUID();
+
+        assertEquals(3, client.sadd(setKey1, new String[] {"1", "2", "3"}).get());
+        assertEquals(2, client.sadd(setKey2, new String[] {"2", "3"}).get());
+        // move an elem
+        assertTrue(client.smove(setKey1, setKey2, "1").get());
+        assertEquals(Set.of("2", "3"), client.smembers(setKey1).get());
+        assertEquals(Set.of("1", "2", "3"), client.smembers(setKey2).get());
+        // move an elem which preset at destination
+        assertTrue(client.smove(setKey2, setKey1, "2").get());
+        assertEquals(Set.of("2", "3"), client.smembers(setKey1).get());
+        assertEquals(Set.of("1", "3"), client.smembers(setKey2).get());
+        // move to a new set
+        assertTrue(client.smove(setKey1, setKey3, "2").get());
```

Comment:
So it actually creates a new set if the destination does not exist? Do you want to talk about this in the documentation above?

Reply:
Added